zio-http/jvm/src/test/scala/zio/http/ClientHttpsSpec.scala

```diff
@@ -74,7 +73,7 @@ abstract class ClientHttpsSpecBase extends ZIOHttpSpec {
           ),
         ),
       )
-    } @@ nonFlaky(20),
```

Comment:
This `nonFlaky` was generating too many calls to `https://untrusted-root.badssl.com/`, which was leading to connect timeouts (probably some anti-ddos/ratelimiting mechanism there)